scripts/src/models/applications.ts

```diff
@@ -99,15 +99,22 @@ export class AppOffer extends BaseEntity {
 				.leftJoinAndSelect("app_offer.app", "app")
 				.where("app_offer.appId = :appId", { appId })
 				.andWhere("offer.type = :type", { type })
-				.orderBy("offer.amount", type === "earn" ? "DESC" : "ASC")
-				.addOrderBy("offer.id", "ASC")
+				.orderBy("offer.ordering", "ASC")
 				.getMany();
 			localCache.set(cacheKey, appOffers);
 		}
 
 		return appOffers;
 	}
 
+	public static async generate(appId: string, offerId: string, cap: Cap, walletAddress: string): Promise<AppOffer> {
+		const lastAppOffer = await AppOffer.findOne({ where: { appId }, order: { ordering: "DESC" } });
+		const orderingBufferStep = 10;
+		const lastAppOfferOrdering = (lastAppOffer && lastAppOffer.ordering) ? Number(lastAppOffer.ordering) : 0;
+		const newAppOfferOrdering = lastAppOfferOrdering + 1 * orderingBufferStep;
```

Comment:
I think "OrderIndex" is a better suffix than Ordering, eg:
`newAppOfferOrderIndex`
(This applies to all relevant var names in this context)